handlers.go

```diff
@@ -1,85 +1,67 @@
-
 package main
+
 import (
-	"net/http"
-	"github.com/gorilla/mux"
-	"log"
 	"encoding/json"
+	"github.com/gorilla/mux"
 	"io/ioutil"
+	"net/http"
 )
 
-// TODO add message fields
-
-type StatusResponse struct {
-    Task_id string `json:"task_id"`
-    Status int `json:"status"`
-}
-
-type StartResponse struct {
-    Task_id string `json:"task_id"`
-}
-
-type StopResponse struct {
-    Task_id string `json:"task_id"`
-		ExitCode int `json:"exit_code"`
-}
-
-// path param task_id : identifier of task query
-// returns status of running service
+// StatusHandler returns status of running service.
 func StatusHandler(w http.ResponseWriter, r *http.Request) {
 
 	vars := mux.Vars(r)
 	task_id := vars["task_id"]
 
-	status := GetProcessStatus(task_id)
-	log.Printf("task_id=%s, status=%d", task_id, status)
+	res := GetProcessStatus(task_id)
 
-	responseBody := StatusResponse{task_id, status}
 	w.WriteHeader(200)
```

Comment:
This should not default to `http.StatusOK` (also use status codes defined the by package instead of raw numeric value as these are hard to read unless memorized) and handle different cases of errors (like an invalid `taskID` which should return `http.StatusNotFound`)